extensions/import/src/wizard/flatFileWizard.ts

```diff
@@ -33,7 +33,6 @@ export class FlatFileWizard {
 		let pages: Map<number, ImportPage> = new Map<number, ImportPage>();
 
 
-		// TODO localize this
```

Comment:
we should still localize this error message

Reply:
Whoops I removed that thinking I had.